### PR TITLE
add example spechbubble

### LINKED ISSUE #16: add speech bubble and text in component

### PR DESCRIPTION
**Pull request by mewiemers**
add example speech bubble in storybook

### CODE REVIEW

File: components/speechbubble/Speechbubble.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styles from "./Speechbubble.module.css";
+
+export default function Speechbubble() {
+  return (
+    <>
+      <div className={styles.bubble}>
+        <p className={styles.text}>
+          Hello (Name)! Nice to meet you! I am Albert and i love to travel!
+          Would you come along with me through Australia?
```
Comment (DNaunin) [inline components/speechbubble/Speechbubble.tsx]: Maybe use "Will you come along with me" or "Would you like to come along with me", the conjugal form sounds not right here. 🐼

**Review by DNaunin — APPROVED** [review]
Great Job!

File: components/speechbubble/Speechbubble.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply components/speechbubble/Speechbubble.tsx]: I agree!

File: components/speechbubble/Speechbubble.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import styles from "./Speechbubble.module.css";
+
+export default function Speechbubble() {
+  return (
+    <>
+      <div className={styles.bubble}>
+        <p className={styles.text}>
+          Hello (Name)! Nice to meet you! I am Albert and i love to travel!
```
Comment (SilviaBarcellini) [inline components/speechbubble/Speechbubble.tsx]:
```suggestion
          Hello (Name)! Nice to meet you! I am Albert and I love to travel!
```

**Review by SilviaBarcellini — APPROVED** [review]
Good job! 💯

File: components/speechbubble/Speechbubble.module.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.bubble {
+  width: 22rem;
+  height: 15rem;
+  background-color: rgb(10, 214, 61);
```
Comment (phlgr) [inline components/speechbubble/Speechbubble.module.css]: This could be a variable

File: components/speechbubble/Speechbubble.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import styles from "./Speechbubble.module.css";
+
+export default function Speechbubble() {
```
Comment (phlgr) [inline components/speechbubble/Speechbubble.tsx:3]: Maybe make the text variable by giving it to the component via a prop

**Review by phlgr — CHANGES_REQUESTED** [review]
GJ 🚀

**Pull request merged**